configs/sst_gpu-unwanted.yaml

```diff
@@ -5,64 +5,11 @@ data:
   description: GPU - Packages which should not be in ELN
   maintainer: sst_gpu
```

Comment:
Is there still a `sst_gpu`?

Reply:
Yep, and we're now (temporarily) our own new group: GPU & Accelerators .